### PR TITLE
Add sessions seed data

### LINKED ISSUE #5535: Adding sessions seed data for development
**Parent Issue**
#4477 

**Is your feature request related to a problem? Please describe.**
According to the parent issue 4477, this aims to add seed data of sessions for development

**Describe the solution you'd like**

adding seed data for sessions

**Additional context**

i'm working on it

### PR DESCRIPTION
**Pull request by prabhupant**
Fixes #5535 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [ ] The unit tests pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [x] All the functions created/modified in this PR contain relevant docstrings.

#### Short description of what this resolves:
 adds seed data for sessions
 
#### Changes proposed in this pull request:

- adds seed data for development of sesions
-
-

### CODE REVIEW

File: tests/hook_main.py [hunk 1]
```diff
@@ -1977,7 +1977,7 @@ def session_track_get_detail(transaction):
     :return:
     """
     with stash['app'].app_context():
-        session = SessionFactory()
+        session = SessionFactory(track_id=1)
```
Comment (srv-twry) [inline tests/hook_main.py]: Why can't we define `track_id` variable defined inside the `SessionFactory` with value 1? Why do we need to pass it as kwarg?

Comment (srv-twry) [reply tests/hook_main.py]: Yes, for all of them.

Comment (prabhupant) [reply tests/hook_main.py]: Alright. Should I make the seed data for event first?

Comment (srv-twry) [reply tests/hook_main.py]: @prabhupant What I meant was that I think there is no need to pass `track_id` variable to the `SessionFactory`.
It is already defined as 1 in the factory here https://github.com/fossasia/open-event-server/blob/c8bfed7bc4ae817a6b775cc81098e1fa6925ffa1/app/factories/session.py#L39

Comment (prabhupant) [reply tests/hook_main.py]: @srv-twry i made the changes you asked for, but now the hooks are timing out. please help

Comment (srv-twry) [reply tests/hook_main.py]: If the hooks were not getting timed out earlier then revert back to what you were doing before. I'll need some time to investigate why it is timing out but currently not in a position to devote that much time(multiple interviews lined up one after the other). So kindly follow what has been done in other places in the project.

**Comment by mariobehling** [comment]
Closing this due to inactivity. Would be great to get this in. Please open a new PR if you are interested to add this. Thanks.

**Pull request closed without merging**